pages/jobs.tsx

```diff
@@ -197,16 +197,10 @@ const Jobs: NextPage<JobsPageProps> = ({ preview, jobs, content }) => {
 export default Jobs;
 
 export async function getStaticProps({ preview = false }) {
-    try {
-        const jobs = await getAllJobSummaries();
-        const content = (await getJobListingPage(preview)) ?? [];
-        return {
-            props: { preview, jobs, content },
-            revalidate: 60 * 60, // After one hour, the cache expires and the page gets rebuilt.
-        };
-    } catch (error) {
-        return {
-            notFound: true,
-        };
-    }
+    const jobs = await getAllJobSummaries();
+    const content = (await getJobListingPage(preview)) ?? [];
+    return {
+        props: { preview, jobs, content },
+        revalidate: 60 * 60, // After one hour, the cache expires and the page gets rebuilt.
+    };
```

Comment:
**Praise**: This is a good update as the `jobs` page should never return a 404, rendering the old job listings is a decent fallback. It means that some listings could appear open for longer than expected, but this is low risk & low impact, as a an applicant wouldn't be able to submit an application past the deadline set on PeopleHR anyway.